zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/encoding/AdminRequest.java

```diff
@@ -80,7 +89,8 @@ public void setKey(final long key) {
     this.key = key;
   }
 
-  public void setType(final AdminRequestType type) {
-    this.type = type;
+  public void setConfiguration(final String configuration) {
```

Comment:
:x: This is a generic `AdminRequest` that shouldn't know about "configuration". The right naming would be `payload` IMO.